docs/styleguide/styles.html

```diff
@@ -2,19 +2,19 @@
 
 {% block content %}
 
-<section class="docs-section">
+<section class="docs-section" id="colors">
 
```

Comment:
Colors, Type and spacing are not clickable navigation nodes. Do we need an id here?